dryplants/juncus.lua

```diff
@@ -105,8 +105,8 @@ biome_lib.register_on_generate({
 		"sumpf:peat",
 		"sumpf:sumpf"
 	},
-    max_count = JUNCUS_NEAR_WATER_PER_MAPBLOCK,
-    rarity = 101 - JUNCUS_NEAR_WATER_RARITY,
+    max_count = 70,
```

Comment:
Shouldn't we create settings for this using `settingtypes.txt`, just for compatibility?

Reply:
I don't think the extra effort is justified by the number of users who use the settings

Reply:
> Shouldn't we create settings for this using `settingtypes.txt`, just for compatibility?

no. see above

Reply:
besides, once converted off biome lib, the settings would have been most likely removed anyways